src/printapp/printstatus.py

```diff
@@ -166,6 +170,7 @@ def fetch_data(self):
         for i in range(numberOfJobs):
             j = i * 7
```

Comment:
Can we change `i` to `job_index` and `j` to something descriptive. Perhaps `job_column`? I'm actually not sure what `j` is actually doing.